utils/ratlib.py

```diff
@@ -128,3 +165,75 @@ def try_parse_uuid(suspect: str) -> UUID:
 
     else:
         return result
+
+
+# color functions
+def color(text: str, text_color: Colors, bg_color=Optional[Colors]) -> str:
```

Comment:
I believe you will find that is not how `Optional` is supposed to be used.
In python, `Optional` is part of the type hint, not the value 😉 
`color`'s signature should look something like
```py
def color(text: str, text_color: Colors, bg_color:Optional[Colors] = None) -> str:
    ...
```

Reply:
...doh. Will fix.